Admin/Custom.Xperience.Stripe.Admin/Constants/XperienceStripeConstants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Custom.Xperience.Stripe
```

Comment:
(Commenting here because I can't comment on the related file) The Custom.Xperience.Stripe project has a reference to nuget-README.md which isn't included in the repo and prevents building

Reply:
The PackageReadmeFile property in csproj was also causing problems, because the NuGet is configured to package on build, but the README doesn't exist